src/components/components/ebay-tooltip-base/index.marko

```diff
@@ -7,12 +7,16 @@ static function toJSON() {
     }
 }
 
+static const isBrowser = typeof window !== "undefined";
 $ input.toJSON = toJSON;
 
 <span
     key="container"
     onExpander-expand("handleExpand")
     onExpander-collapse("handleCollapse")
     overlay-style=input.overlayStyle>
+    <!-- <if(isBrowser)>
```

Comment:
Are the changes in this file still needed?

Reply:
nope I will drop those...